cursive-core/src/printer.rs

```diff
@@ -118,16 +115,53 @@ impl<'a, 'b> Printer<'a, 'b> {
         }
     }
 
+    /// Prints some text at the given position.
+    ///
+    /// # Parameters
+    /// `start` is an offset used to print the text in the view.\
+    /// `text` is a simple string to print on the screen.
```

Comment:
```suggestion
    /// * `start` is the offset of the first character to print.
    /// * `text` is a simple string to print on the screen.
```

Reply:
The first char is not the only one affected by the `start` parameter. This parameter affect the whole `text`.